kf_lib_data_ingest/etl/configuration/target_api_config.py

```diff
@@ -9,8 +9,8 @@ class Foo:
         class_name = 'foo'
         target_id_concept = CONCEPT.FOO.TARGET_SERVICE_ID
 
-        @staticmethod
-        def transform_records_list(records_list):
+        @classmethod
```

Comment:
Do we have use cases where these methods need to be factories for the class?

Reply:
Not factories, but some of the methods do refer to the class itself (not all, though) and I wanted to switch to using `cls` instead of the specific declaration because I felt like it added some visual clarity.

Example:
Visually differentiating between
`"kf_id": get_target_id_from_record(cls, record),`     <- self class
and 
`"investigator_id": get_target_id_from_record(Investigator, record),`   <- other class
on https://github.com/kids-first/kf-lib-data-ingest/pull/513/files/a4cb66e73a110d852f66b5cfa63b3276438e48bb#diff-443b5b7d88394f11f7ccf7a8fc95383d265626dcaedb88e62d2043f7c0384248R89-R90